README.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
Something is wrong with the headings in this file (too many levels?), so the end of the file doesn't render nicely (https://github.com/arfc/predicting-the-past/blob/375cb332036289a55af0671dd3579a7520b43c68/README.ipynb)